lib/internal/process/pre_execution.js

```diff
@@ -735,7 +735,8 @@ function runEmbedderPreload() {
 function loadPreloadModules() {
   // For user code, we preload modules if `-r` is passed
   const preloadModules = getOptionValue('--require');
-  if (preloadModules && preloadModules.length > 0) {
+  const isOrcastrationProcess = getOptionValue('--test');
```

Comment:
Typo in `Orchestration`, and I think the name can be more explicit:
```suggestion
  const isTestRunnerOrchestrationProcess = getOptionValue('--test');
```